docs/getting-started/quick-start.md

```diff
@@ -4,12 +4,152 @@ sidebar_position: 2
 
 # Quick Start
 
-In this section we are providing a sample application that one can deploy and just use to play around.
+In this quick start, you will see a sample application that you can deploy and
+use to play around.  As part of this exercise, a backing service and an
+application is required.  The backing service is a PostgreSQL database and
+application is a [Spring Boot REST API server][petclinic].
 
-## K8S
+Here is a schematic diagram of this setup:
 
-_Here we provided a small sample on K8s_
+![postgresql-spring-boot](/img/docs/postgresql-spring-boot.png)
 
-## OpenShift
+The service binding operator will collect backing service configuration required
+for connectivity and expose it to the applications.
 
-_Here we provided a small sample on OpenShift_
\ No newline at end of file
+## Prerequisites
+
+Ensure you have installed and configured these:
+
+- Kubernetes cluster (Use [minikube](https://minikube.sigs.k8s.io/) or
+  [kind](https://kind.sigs.k8s.io/) locally)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+- [Service Binding Operator](installing-service-binding).
+
+## Database Backend
+
+The application is going to connect to a PostgreSQL database backend.  The
+PostgreSQL can be setup using the [Crunchy PostgreSQL operator from
+OperatorHub.io][crunchy].
+
+The installation of the operator doesn't create a database instance for
+connection.  To create a database instance, you need to create custom resource
```

Comment:
Suggestion: Please put these instructions as steps. Steps for connecting application to a database backend.

Put the heading for this section as "Creating a database instance" and link this section to the 2nd bullet item mentioned in the 1st paragraph under "Quick start". I will review again after the rewrite.

Reply:
Done. 34a6d23f94033d82bb896681fab523ea9dc5b9a0